### DISCUSSION TITLE
local user inside image (/home not preserved)

### DISCUSSION THREAD

**Discussion by Soylor** [ltsp/ltsp#795] [General] [answered]
I've got a jammy image with 2 users created (user 1 and user2)
user 1 home is /home/user1 and user2 home is /home/user2 (both hone/user1 and home/user2 are created directories)

I created image with --cleanp=0 and PWMERGE_DUR="user2"

I got login in with user2 home directory is / and /home is quite empty.

I cannot preserve /home structure inside image?
Thanks.

**Comment by alkisg** [comment]
No.
The image is read only (for the root / dir) and LTSP creates a tmpfs overlay over it to store the changes.
The /home directory can't be read only, so it cannot come from the image.

**Comment by Soylor** [comment, marked as answer]
Got it working creating the user home at runtime wiith a POST_SERVICE_x script (Creates home for user2 with permissions).
This way i get a NON persistent local user (everything is deleted after reboot).
